benchmark/loading/PODLib.dat

```diff
@@ -0,0 +1,75 @@
+Load time PODLib / alkyl 1.8571031093597412
```

Comment:
Is there a reason why you save `Load time PODLib /` looks like unnecessary space consumption.